packages/server/src/adapters/standalone.ts

```diff
@@ -41,18 +36,5 @@ export function createHTTPServer<TRouter extends AnyRouter>(
   opts: CreateHTTPHandlerOptions<TRouter>,
 ) {
   const handler = createHTTPHandler(opts);
-  const server = http.createServer((req, res) => handler(req, res));
-
-  return {
-    server,
-    listen: (port?: number) => {
-      server.listen(port);
-      const actualPort =
-        port === 0 ? ((server.address() as any).port as number) : port;
-
-      return {
-        port: actualPort,
-      };
-    },
-  };
+  return http.createServer((req, res) => handler(req, res));
```

Comment:
this is now an extremely slim wrapper over the built-in. is it worth having it and not just mention in docs how to use createHTTPHandler with node.httpServer?

Reply:
In the perfect world yes, but it’d also break all current users of standalone adapter. Maybe we can soft-deprecate it in v11 and remove in v12?

I wonder if there might be some future improvements we’d like to add directly in here though 🤔

Reply:
This is the easiest way for a newb to stand up a tRPC server, and will work for most deployments, so I think it's worth keeping. When it comes to documentation alone this is the best starting point, and since it's such a skinny wrapper it's no maintenance problem really!

Reply:
Yea ik and I like having this simple option! But we do keep a handful of test for it that could be removed(?).

But yea we shouldn't just remove it without first deprecating it. Let's make this change and we can continue the discussion later